app/Config/Schema/schema.xml

```diff
@@ -1109,6 +1109,7 @@
     </field>
     <field name="introduction_text" type="X" size="4000" />
     <field name="conclusion_text" type="X" size="4000" />
+    <field name="introduction_text_pf" type="X" size="4000" />
```

Comment:
Maybe use the step name so it's clearer? `introduction_text_petitioner_attributes`, or if that's too long `introduction_text_pa`?